duplicate.php

```diff
@@ -26,17 +26,20 @@ function edd_duplicate_product() {
 }
 
 /**
- * Get a product from the database
+ * Gets a product from the database.
+ *
+ * @param int|string $id The ID of the download to duplicate.
+ * @return object|null Returns an object if a download is found; otherwise null.
  */
-function edd_get_product_to_duplicate($id) {
+function edd_get_product_to_duplicate( $id ) {
 	global $wpdb;
 
 	$post = $wpdb->get_results( "SELECT * FROM $wpdb->posts WHERE ID=$id" );
```

Comment:
Since we only want one value, we could make this only return one value. That would remove the need for the array `reset()` further down.

```suggestion
	$post = $wpdb->get_row( "SELECT * FROM $wpdb->posts WHERE ID=$id" );
```

But if we change this then we have to make changes to the if statement below, as that would actually start working.

Reply:
I actually did end up making this same change in #15 because querying by ID will be just one row, never many. That PR is branched off of this one because of the overlap here--might be better to review that one first, which I should have told you earlier. Sorry!